cherrypy/test/test_bus.py

```diff
@@ -218,6 +220,7 @@ def f(method):  # pylint: disable=invalid-name
         assert bus.state in states, 'State %r not in %r' % (bus.state, states)
 
 
+@pytest.mark.xfail(CI_ON_MACOS, reason="continuous integration on macOS fails")
```

Comment:
Does this happen in any macOS env or only in Circle CI?
I've restarted macOS jobs @ https://travis-ci.org/github/cherrypy/cherrypy/builds/743540522 to figure out if it's also true for Travis CI.

Reply:
Confirmed. This is also happening on Travis CI: https://travis-ci.org/github/cherrypy/cherrypy/jobs/743540538#L17898.
I was worried that the condition may be not precise enough and would xfail unrelated envs but it seems it's okay.